scripts/build/common.mjs

```
@@ -54,6 +54,24 @@ export const banner = {
 `.trim()
 };
 
+/**
+ * @param {import("esbuild").BuildOptions[]} buildConfigs
+ */
+export async function buildOrWatchAll(buildConfigs) {
+    if (watch) {
+        await Promise.all(buildConfigs.map(cfg =>
+            context(cfg).then(ctx => ctx.watch())
+        ));
+    } else {
+        await Promise.all(buildConfigs.map(cfg => build(cfg)))
+            .catch(error => {
+                console.error(error.message);
+                // esbuild will already print the error, so just exit and swallow the error (to avoid ugly stacktrace)
```

```suggestion
                // esbuild will already print the error, so just exit and swallow the error (to avoid ugly stacktrace)
```

This comment seems kinda weird now that you are printing the error message. Stack trace is still being ignored but it's not like we are completely swallowing the error